modules/os2forms_organisation_openid_connect/os2forms_organisation_openid_connect.info.yml

```diff
@@ -6,3 +6,4 @@ core_version_requirement: ^9
 
 dependencies:
   - os2forms_organisation:os2forms_organisation
+  - os2forms_nemlogin_openid_connect:os2forms_nemlogin_openid_connect
```

Comment:
Is anything from the `os2forms_nemlogin_openid_connect` module used in this module?

Reply:
Removed.